interpreter/python/python.go

```diff
@@ -655,32 +656,40 @@ func (d *pythonData) readIntrospectionData(ef *pfelf.File, symbol libpf.SymbolNa
 
 // decodeStub will resolve a given symbol, extract the code for it, and analyze
 // the code to resolve specified argument parameter to the first jump/call.
-func decodeStub(ef *pfelf.File, addrBase libpf.SymbolValue, symbolName libpf.SymbolName,
-	argNumber uint8) libpf.SymbolValue {
-	symbolValue, err := ef.LookupSymbolAddress(symbolName)
+func decodeStub(
+	ef *pfelf.File,
+	memoryBase libpf.SymbolValue,
+	symbolName libpf.SymbolName,
+) libpf.SymbolValue {
+	codeAddress, err := ef.LookupSymbolAddress(symbolName)
 	if err != nil {
 		return libpf.SymbolValueInvalid
 	}
 
 	code := make([]byte, 64)
-	if _, err := ef.ReadVirtualMemory(code, int64(symbolValue)); err != nil {
+	if _, err := ef.ReadVirtualMemory(code, int64(codeAddress)); err != nil {
 		return libpf.SymbolValueInvalid
 	}
+	dumpCode := func() {
+		log.Debugf("python stub code: %s", hex.Dump(code))
```

Comment:
How about changing `debugStub` to return also an `error` and put the dump there?